<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartingDialog</class>
 <widget class="QDialog" name="StartingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>1000</width>
     <height>800</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/res/res/begining.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_localpvp">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>310</y>
     <width>200</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/res/localpvp.png</normaloff>:/res/res/localpvp.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pve">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>200</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/res/pve.png</normaloff>:/res/res/pve.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_editor">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>470</y>
     <width>200</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/res/editor.png</normaloff>:/res/res/editor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/res/res/exit.png</normaloff>:/res/res/exit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>StartingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>668</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
